engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/bool/AndNode.java

```diff
@@ -13,7 +14,8 @@
 @BuiltinMethod(
     type = "Boolean",
     name = "&&",
-    description = "Computes the logical conjunction of two booleans")
+    description = "Computes the logical conjunction of two booleans",
+    inlineable = true)
```

Comment:
`inlineable` is `true` by default. So why add it? Same in other cases in this PR

Reply:
`inlineable` is a tri-state attribute. More at https://github.com/enso-org/enso/pull/6442/files#r1180249274